NEWS.md

```diff
@@ -14,6 +14,7 @@
 * Removed `use_try_from` as an option in `rust_function`, and added `use_rng` (#354)
 * Added `use_crate()` function to make adding dependencies to Cargo.toml easier within R, similar to `usethis::use_package()` (#361)
 * Fixed an issue in `rust_source()` family of functions that prevented usage of `r#` escape sequences in Rust function names (#374)
+* `use_cran_defaults()` now checks the `systemRequirements` field in the `DESCRIPTION` file for cargo and rustc. It will display installation instructions if either is missing or provide the minimum required version if the installed version is outdated.
```

Comment:
```suggestion
* `use_cran_defaults()` now checks the `SystemRequirements` field in the `DESCRIPTION` file for cargo and rustc. It will display installation instructions if either is missing or provide the minimum required version if the installed version is outdated.
```

Reply:
If we can resolve this, I think it looks good to merge! 
We can address the cargo and rustc messages separately.